crates/compiler/builtins/roc/Bool.roc

```diff
@@ -6,7 +6,7 @@ interface Bool
 ##
 ## Total equality means that all values of the type can be compared to each
 ## other, and two values `a`, `b` are identical if and only if `isEq a b` is
-## `Bool.true`.
+## `Bool.true`.test
```

Comment:
Is this meant to be here?

Reply:
Oh, sorry I was working on the CI issue and testing if the action is triggering or not? I'll remove it and update the PR